backend/src/main/java/de/amos/apachepulsarui/dto/SubscriptionDto.java

```diff
@@ -48,14 +57,18 @@ public static SubscriptionDto create(SubscriptionStats subscriptionStats, List<M
                 .allConsumers(consumers)
```

Comment:
Maybe instead of passing all and active consumers, we could pass active and inactive ones?